src/ngrx/4-create-actions/create-actions.md

```diff
@@ -1,4 +1,64 @@
 @page learn-ngrx/create-actions Creating Actions
 @parent learn-ngrx 4
 
-@description Learn how to create NgRx actions.
\ No newline at end of file
+@description Learn how to create NgRx actions.
+
+## Overview
+
+TODO
+
+## Problem 1: Create Login Actions to Represent Login Events
+
+NgRx Schematics generated Actions with the following Action Types: `[Login] Load Logins`, `[Login] Load Logins Success` and `[Login] Load Logins Failure`.
+
+Our goal is to have a different set of Action instead:
+
+- `login` - dispatched when user submits username and password
+    - type: `[Login Page] Login`
+    - props: `{ username: string, password: string }`
+- `loginSuccess` - dispatched by an Effect when user has successfully logged in
+    - type: `[Login API] Login Success`
+    - props: `{ userId: string; username: string; token: string }`
+- `loginFailure` - dispatched by an Effect when user login attempt fails
+    - type: `[Login API] Login Failure`
+    - props: `{ errorMsg: string }`
+- `logout` - dispatched when user clicks on a logout button
+    - type: `[Chat Page] Logout`
```

Comment:
we might want to update this logout description from Chat Page to something else that represents the final MVP. In the future, when we have an actual chat, we change it back

Reply:
Also the action type has already been updated and there's 2 new ones: https://github.com/bitovi/angular-ngrx-chat/compare/ngrx-init...create-actions#diff-4f963ea9a3ecddf4980d73239bbf41aa99b347895befdb1f81cba63392c41b72R18